Update Images and Documentation

ryan-van opened the pull request:
Hey, I updated the documentation and images with the new finance for issue #30 in finance-activity.

quozl commented:
Thanks.  I've quickly reviewed;

* [ ] the old images are not removed,
* [ ] the image file type is PNG (qty 4) instead of png (qty 577),
* [ ] the images are not 4:3 aspect ratio,
* [ ] the images are too large; aim for around 640x480 after resizing, because they should display inline using the Help activity on a typical child's laptop, such as 1366x768,
* [ ] the addressed person keeps changing; (we, you, students),
* [ ] the tense might need changing, not sure,
* [ ] _you're spendings_ should be _your spending_,

You may also be interested in the test data set built into the Finance activity.  To display that test data, start the activity from the Terminal activity like this;

```
cd ~/Activities/Finance.activity
FINANCE_TEST=true sugar-activity3
```
and then switch to Budget view before switching back to Register view.

Also see Wikipedia's diagram of [common display resolutions](https://en.wikipedia.org/wiki/Display_resolution).  16:9 aspect ratio is the lowest cost at the moment.

ryan-van commented:
Hey, seems I'm making tons of simple mistakes everywhere. I hope I cleaned it up well this time.

- Old images removed, new ones using test data
- the image file type converted from PNG to png
- the images are now 4:3 aspect ratio w/ resolutions of 640 x 480
- change first/second person pronouns to all third person
- fixed tense conflicts of present/past to all present tense.

Let me know if I'm missing anything else!
